### ISSUE TITLE
Server crashed on delete comment

### ISSUE THREAD

**Issue by Tereseus** [justrobd/project-3-the-scoop#3]
So as you can see below I was unable to delete the comment I had saved because the id for that comment is undefined. Are you able to reproduce this error on your end? There could be multiple reasons for this issue, I checked your util files and views and everything seemed fine. There is some error going on in the logic that is preventing it from finding the id. Further investigation will be needed for us to fix this issue.

![image](https://user-images.githubusercontent.com/19315896/36829884-1cdc10d8-1cef-11e8-90d5-3f1926bec0cb.png)

**Comment by justrobd** [comment]
Yes, I can replicate it. If I add a comment then delete that comment. Then add another comment and try to delete that comment. That is when the server crashes with the same error you are getting. 
![screenshot from 2018-03-01 10-13-41](https://user-images.githubusercontent.com/4628352/36855734-e6094c7e-1d39-11e8-8d71-a71dda870f37.png)
